providers/gupshup/package.json

```diff
@@ -18,7 +18,7 @@
     "fix:lint": "eslint src --ext .ts --fix",
     "test": "run-s build test:*",
     "test:lint": "eslint src --ext .ts",
-    "test:unit": "echo 'Tests Missing'",
+    "test:unit": "jest src",
```

Comment:
@scopsy I recovered this unit test that was ignore here: https://github.com/novuhq/novu/pull/956/files#diff-b4f5916d556c469da590e5d8ff33137169060f70c6c90b9818d041fc3cec48f0R21
let me know if it's good.